src/components/SectionCard.js

```diff
@@ -0,0 +1,17 @@
+import CircleGauge from './CircleGauge';
+
+function SectionCard({ section, percentage, subcategories, color }) {
+  return (
+    <div className="bg-white shadow-widgit flex flex-nowrap justify-around">
```

Comment:
We could add a `rounded` class here to round the borders.

Reply:
Actually, there is an extension on the rounded class we can use, which should be `rounded-widgit`